src/Datadog.Trace/TracerSubProcessManager.cs

```diff
@@ -10,40 +11,74 @@ namespace Datadog.Trace
 {
     internal class TracerSubProcessManager
```

Comment:
Can we rename this to `ProcessManager` or even `ProcessManager`?

Reply:
You don't think this describes what it does well?

Reply:
Most of the rename suggestions only apply if we make this a general process manager. Except for changing `Sub` to `Child` everywhere because "child process" is the usual term in computing.